src/getConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { exec } from 'child_process';
+
+interface Config {
+  project: {
+    android: {
+      packageName: string;
+    }
+  }
+}
```

Comment:
In `cli.ts:130` we have line:

```ts
dstPath = dstPath || ctx.root;
```

But according to `getConfig` it may return `null`, and TS is complaining about that (which prevents compiling the CLI file).

I substituted it as:

```ts
dstPath = dstPath || ctx?.root || '.';
```

And it worked, but maybe there is better solution?

cc @hannojg

Reply:
yeah, I think thats a fair change. Do you agree @szymonrybczak

Reply:
yea, sure!